Examples/exemple.txt

```diff
@@ -0,0 +1,6 @@
+if 1 : (X <= 0)
+	then 2 : X := (0 - X)
+	else 3 : X := (1 - X);
+if 4 : (X=1)
+	then 5 :X:=1
+	else 6 :X :=(X+1)
```

Comment:
Just need to add `0 : X := some_number;`at the beginning to allow the eval of the program, but I will do it in another PR